docs/source/install-kube.rst

```diff
@@ -216,9 +216,9 @@ There are no other requirements for images, any image that meets the above
 should work fine. You may install any additional libraries or dependencies you
 require.
 
-To develop your own image, you may either base it on a compatible version of
-``daskgateway/dask-gateway``, or use our `example dockerfile`_ as a reference
-and develop your own.
+We encourage you to maintain your own image for scheduler and worker pods as
+this project only provides a `minimal image`_ for testing purposes which can be
+used as inspiration, but isn't recommended to be used as a base image.
```

Comment:
```suggestion
We encourage you to maintain your own image for scheduler and worker pods as
this project only provides a `minimal image`_ for testing purposes.
```

I made this suggestion based on the discussion just above in https://github.com/dask/dask-gateway/pull/533/files#r852284873.

Reply:
ok